### PR TITLE
Create client portal messages

### PR DESCRIPTION
**Pull request by bytheway875**
Screen Shot 2021-04-13 at 2 00 52 PM
Screen Shot 2021-04-12 at 5 47 14 PM

### CODE REVIEW

File: app/helpers/contact_record_helper.rb [hunk 1]
```diff
@@ -16,6 +16,8 @@ def message_heading(contact_record)
       "#{I18n.t("hub.messages.from")} #{contact_record.from}"
     when :outgoing_email
       "#{I18n.t("hub.messages.to")} #{contact_record.to}"
+    when :incoming_portal_message
```
Comment (embarnard) [inline app/helpers/contact_record_helper.rb:19]: is it obvious that this is from the client if it just says from the client portal? Maybe redundant but would also mimic the structure of the other messages if included their name

**Review by embarnard — APPROVED** [review]
Looks good! No blocking comment

File: app/views/portal/portal/home.html.erb [hunk 2]
```diff
@@ -92,8 +92,12 @@
...
-    <div class="spacing-above-60">
+  <% end %>
+
+  <div class="spacing-above-60">
+    <% if @tax_returns.present? %>
       <%= link_to t(".submit_documents"), portal_upload_documents_path, class: "button button--wide" %>
-    </div>
-  <%end %>
+    <% end %>
+    <%= link_to t(".send_message"), new_portal_message_path, class: "button button--wide button--primary" %>
```
Comment (embarnard) [inline app/views/portal/portal/home.html.erb:101]: the placement of the button might change a little with the changes from this refactoring branch that was merged https://github.com/codeforamerica/vita-min/pull/1007

File: app/helpers/contact_record_helper.rb (hunk 1 above)
Comment (bytheway875) [reply app/helpers/contact_record_helper.rb:19]: It will include their name! See screenshot. This is just what gets appended after the name.

**Pull request merged**